src/protvista-uniprot-structure.ts

```diff
@@ -140,6 +149,15 @@ class ProtvistaUniprotStructure extends LitElement {
     protvistaDatatableElt.selectedid = this.pdbId;
   }
 
+  addStyles() {
+    // We are not using static get styles()
+    // as we are not using the shadowDOM
+    // because of Mol*
+    const styleTag = document.createElement('style');
+    styleTag.innerHTML = loaderStyles.toString();
+    document.querySelector('head')?.append(styleTag);
```

Comment:
Could it ever happen where styles are appended multiple times to the `head`?

Reply:
Related to https://github.com/ebi-webcomponents/protvista-uniprot/pull/35#discussion_r660408723